x/feegrant/types/types.proto

```diff
@@ -0,0 +1,101 @@
+syntax = "proto3";
```

Comment:
What kind of IDE do you use? If you use VSCode or something similar, can you install the `clang-format` extension? Then the following config will allow you to have cleaner formatting that is consistent with the rest of the framework:

```json
"clang-format.style": "{BasedOnStyle: Google, IndentWidth: 2, ColumnLimit: 120, AlignConsecutiveAssignments: true, AlignConsecutiveDeclarations: true, SpacesInSquareBrackets: true}"
```

Reply:
I use GoLand. Added the extension now, thanks